### PR TITLE
Add --all flag to include subdependencies

### PR DESCRIPTION
**Pull request by 3dteemu**
`npm ls --prod --json` only lists top level dependencies.
- Add `--all` to include subdependencies
- Use `--omit=dev` instead of `--prod` per npm suggestion

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "datwd",
-  "version": "0.2.0",
+  "version": "0.2.1",
```
Comment (kmjennison) [inline package.json:3]: Please remove this version change. I'll bump it on main when I release.

File: yarn.lock
Comment (kmjennison) [inline yarn.lock:1]: Please revert changes to yarn.lock.

**Review by kmjennison — CHANGES_REQUESTED** [review]
Thanks for the PR! Just a few small change requests, and then I'll get this merged.

File: package.json (hunk 1 above)
Comment (3dteemu) [reply package.json]: Done!

File: yarn.lock
Comment (3dteemu) [reply yarn.lock:1]: Done!

**Review by kmjennison — APPROVED** [review]
Thanks!

File: src/index.js [hunk 2]
```diff
@@ -19,7 +19,7 @@ const getAllDependencies = (moduleFilterInput = []) => {
 
   // Get the full dependency tree using NPM, excluding dev dependencies
   // and peer dependencies.
-  const dependencyTree = JSON.parse(execSync('npm ls --prod --json').toString())
+  const dependencyTree = JSON.parse(execSync('npm ls --all --omit=dev --json').toString())
```
Comment (kmjennison) [inline src/index.js:22]:
```suggestion
  const dependencyTree = JSON.parse(
    execSync('npm ls --all --omit=dev --json').toString()
  )
```

**Pull request merged**